apps/web/src/views/GaugesVoting/components/Table/GaugesTable/index.tsx

```diff
@@ -50,6 +46,30 @@ export const GaugesTable: React.FC<
     setSortBy(by)
   }
 
+  const Row = useCallback(
+    ({ data: rows, index, style }): JSX.Element => {
+      const row = rows[index]
+      return (
+        <TableRow
+          style={style}
+          data={row}
+          locked={selectRows?.find((r) => r.hash === row.hash)?.locked}
+          selectable={selectable}
+          selected={selectRows?.some((r) => r.hash === row.hash)}
+          onSelect={onRowSelect}
+          totalGaugesWeight={totalGaugesWeight}
+        />
+      )
+    },
+    [onRowSelect, selectRows, selectable, totalGaugesWeight],
+  )
```

Comment:
Maybe we can add `React.memo` to `TableRow` so we don't need this useCallback?

Reply:
@chefjackson done